ada_2_fhir/zibs2017/payload/zib-procedure-2.1.xsl

```diff
@@ -439,7 +449,28 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                         </identifier>
                     </xsl:for-each>
                     
-                    <status value="active"/>
+                    <status value="active">
```

Comment:
Shouldn't this fixed `value` be removed here, and be handled by the logic below?